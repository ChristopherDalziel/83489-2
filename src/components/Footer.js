import React from "react";
import styled from "@emotion/styled";

const FooterDiv = styled.footer`
  margin-top: auto;
  height: 6vh;
  background-color: #e63846;
  border-radius: 50px 0px 0px 0px;
  display: flex;
  align-items: flex-end;
  justify-content: flex-end;
  padding: 2vh;
  font-size: 50px;
`;

const Footer = () => {
  return (
    <FooterDiv>
      <a
        target="_blank"
        href="https://github.com/ChristopherDalziel/83489-2"
        rel="noopener noreferrer"
        class="fab fa-github-square socials"
        id="footerLink"
      >
        <span>Link to projects GitHub repository</span>
      </a>
    </FooterDiv>
  );
};

export default Footer;
