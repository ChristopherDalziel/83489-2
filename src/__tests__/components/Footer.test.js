import React from "react";
import { shallow } from "enzyme";
import Footer from "../../components/Footer";

import toJSON from "enzyme-to-json";

test("Should render footer correctly", () => {
  const wrapper = shallow(<Footer />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
