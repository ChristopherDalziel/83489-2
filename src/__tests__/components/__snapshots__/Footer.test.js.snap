// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render footer correctly 1`] = `
<Styled(footer)>
  <a
    class="fab fa-github-square socials"
    href="https://github.com/ChristopherDalziel/83489-2"
    id="footerLink"
    target="_blank"
  />
</Styled(footer)>
`;
